File: est datos/cpp/creacion_y_recorrido.cpp
#include <iostream>


using namespace std;

int main(int argc, char const *argv[])
{
    int tamaño;
    cout << "De que tamaño desea su arreglo? " << endl;
    cin >> tamaño;
    cin.ignore(); // Limpiar el buffer de entrada

    int arr[tamaño];

    cout << "Vamos a llenar el arreglo" << endl;
for (int i = 0; i< tamaño ; i++){
    cout << "Ingrese el elemento # " << (i+1) << ":" << endl;
    cin >> arr[i];
}

cout << "Los elementos del arreglo son: " << endl;

for (int i =0; i< tamaño ; i++){
    cout << "Elemento #" << (i+1) << ": " << arr[i] << endl;
}


    return 0;
}

File: est datos/c#/Creacion_y_Recorrido.cs
using System;
using System.Runtime.InteropServices;
class program {
    static void Main(string[] args)
    {
        int tam;

        Console.WriteLine("Ingrese el tamaño del arreglo:");
        tam = int.Parse(Console.ReadLine());

        int[] arreglo = new int[tam];

        Console.WriteLine("Ingrese los elementos del arreglo:");

        for (int i = 0; i < tam; i++)
        {
            Console.Write($"Elemento {i + 1}: ");
            arreglo[i] = int.Parse(Console.ReadLine());

        }

        Console.WriteLine("Los elementos del arreglo son:");

        for (int i = 0; i < tam; i++) { 
            Console.WriteLine("Elemento # " + (i+1) + ": " + arreglo[i]);

        }
    }
}